### PR TITLE
bugfix #298: Tenable IO Exception Handling Question

### PR DESCRIPTION
**Pull request by dheerajrp**
# Description

Addressed the bugfix #298 issue raised by @brianp9906

Fixes # (issue)

Fixed InvalidInputError and NameError

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] Pytest

**Test Configuration**:
* Python Version(s) Tested: 3.8

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/io/test_tags.py [hunk 1]
```diff
@@ -1052,4 +1074,5 @@ def test_tags_edit_without_filters(api):
                 api.tags.edit(tag_id)
                 flag = False
         except:
```
Comment (mvalliammal) [inline tests/io/test_tags.py:1076]: Why cant we get an exception name here?

**Review by mvalliammal — COMMENTED** [review]
In test_tags.py, why cant line 6 and line 9 be in the sequence lines?

File: tests/io/test_tags.py (hunk 1 above)
Comment (dheerajrp) [reply tests/io/test_tags.py]: Added the exception

**Review by mvalliammal — APPROVED** [review]
I approved the changes for the tenable IO exception issue resolved , verified

**Comment by mvalliammal** [comment]
@brianp9906, the issue reported is solved, please verify

**Pull request merged**